modules/services/mapillary.js

```diff
@@ -275,24 +339,22 @@ export default {
     // Get visible map (point) features
     mapFeatures: function(projection) {
         const limit = 5;
-        const ret = searchLimited(limit, projection, _mlyCache.points.rtree);
-        return ret;
+        return searchLimited(limit, projection, _mlyCache.points.rtree);
     },
+
     // Get filtered Map (points) features (utility-pole, street-light, bench, bike-rack, fire-hydrant)
     filteredMapFeatures: function(projection) {
```

Comment:
If I recall correctly, `filteredMapFeatures()` no longer gets called anymore. If this is still the case we should delete this method!

Reply:
Looks gewd otherwise- a couple of smol requests/suggestions